dot/node.go

```diff
@@ -40,11 +40,11 @@ var logger = log.NewFromGlobal(log.AddContext("pkg", "dot"))
 
 // Node is a container for all the components of a node.
 type Node struct {
-	Name          string
-	Services      services.ServiceRegistryIFace // registry of all node services
-	wg            sync.WaitGroup
-	started       chan struct{}
-	metricsServer *metrics.Server
+	Name            string
+	ServiceRegister services.ServiceRegisterer // registry of all node services
```

Comment:
nit - sorry if I mis-suggested previously - I think `ServiceRegister` should be `ServicesRegistry` (or `ServicesRegisterer`), otherwise it sounds like a function field or method.

Reply:
Updated to 'ServiceRegistry'.